### DISCUSSION TITLE
Tutorials for beginners

### DISCUSSION THREAD

**Discussion by huberax** [theislab/scvelo#1108] [Q&A]
Hello all,
I am relatively new to scRNAseq analysis and have been learning over the past year. I have analyzed my dataset with Seurat and other programs in R. I was interested in performing velocity analysis on the UMAPs of my control and treated group. 
I have used the 10X Genomics cloud analysis platform to generate my Feature / cell matrix HDF5 (filtered) and Genome-aligned BAM files.
I understand in order to run scvelo I can use my UMAP embeddings from my seurat objects and loom files created from the Bam files. However, I am having trouble finding detailed tutorials showing me how to do all of this. 
I understand that I am probably not the first person to struggle with this, so I wanted to ask this community if they know of any detailed tutorials for beginners that can help me properly process my Bam files to get loom files, and perform scvelo analysis with my loom files and UMAP embeddings from my Seurat objects (From what I have found online, there is quite a bit of content to support me at the point in which I already have my processed loom files and UMAP embeddings, but the steps before are unclear to me).
Thank you,
Alex

**Comment by JWJ13164328557** [comment]
You need to use velocyto，The basic usage is velocyto run -o ./ -m /path/mouse/mus_repeat.gtf   new.bam     ../mouse/genes.gtf
